kotlin/workflow-runtime/src/test/java/com/squareup/workflow/internal/RealRenderContextTest.kt

```diff
@@ -172,8 +173,12 @@ class RealRenderContextTest {
   }
 
   @Test fun `makeEventSink works with OutputT of Nothing`() {
-    val context = RealRenderContext<String, Nothing>(PoisonRenderer(), eventActionsChannel)
-    val sink: Sink<String> = context.makeEventSink { null }
+    @Suppress("UNCHECKED_CAST")
+    val context = RealRenderContext(
+        PoisonRenderer(),
+        eventActionsChannel as SendChannel<WorkflowAction<String, Nothing>>
```

Comment:
Why can't be infer the type here? So many new unchecked casts introduced...

Reply:
Beats me, something to do w/the loss of `out` on `OutputT`, but the compiler wants what the compiler wants.

All the casts are in unit tests of the machinery itself, so I'm not concerned.

Reply:
The casts were all due to laziness, re-using helper stubs whose declared types didn't match the requirements in the specific test case. All fixed.